dataactvalidator/filestreaming/fieldCleaner.py

```diff
@@ -106,10 +106,10 @@ def cleanLength(length):
             raise ValueError("Length must be positive")
         return length
 
-    @staticmethod
-    def cleanRow(row, fileType, validationInterface):
+    @classmethod
+    def cleanRow(cls, row, fileType, interfaces):
         for key in row.keys():
-            field_type = validationInterface.getColumn(key, fileType).field_type.name
+            field_type = interfaces.validationDb.getColumn(key, fileType).field_type.name
```

Comment:
I know it doesn't matter here, but is there a reason we're passing all the interfaces with the `interfaces` parameter vs just the `validationInterface`?